index.js

```diff
@@ -10,6 +10,7 @@ const port = process.env.PORT || 3000;
 
 app.use(cors());
 app.use(express.json());
+app.use(express.static('temp')); // Make PDFs accessible
```

Comment:
all pdfs would be saved to a directory called `temp` then, maybe we could name this something more descriptive?

Reply:
well it's not a final save, it's a temporary save. it deletes it after 30 min. i think temp is a good name because in the future (if we generate charts) we can save other temp files here too! 

My thought is, if we end up needing to expose other static files, we can make a better name, but for now this suffices

Reply:
agree with that!